gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -111,27 +120,98 @@ typedef struct _GstTensorFilterProperties
   GstTensorsInfo output_meta; /**< configured output tensor info */
 
   const char *custom_properties; /**< sub-plugin specific custom property values in string */
-  const char *accl_str; /**< accelerator configuration passed in as parameter */
+  union {
+    const char **supported_accl; /**< accelerators supported by framework intersected with user provided accelerator preference */
+    const char *accl_str; /**< accelerator configuration passed in as parameter */
```

Comment:
Why these two values are in a union? ```supported_accl``` is a value configured by a subplugin and ```accl_str``` is a value configured by a user (pipeline), right?

Reply:
Yes, these two values are in union because v0 uses `accl_str` (current implementation), and v1 will use `supported_accl`.
In v1, `tensor_filter.c` will call `getFrameworkInfo` to get the list of all the supported frameworks. `tensor_filter.c` will intersect this with user passed list of accelerators, and the intersection (`supported_accl`) will be passed to subplugin when opening it. Subplugin does not directly need to know the exact value configured by the user in the pipeline (`accl_str`). 

Regarding dynamically updated property values, I think they can be best handled with sendEvent (`updateProperty`).

Reply:
If a given single version will use only one part of the union, it's ok.

Reply:
done.